docker-compose.yml

```diff
@@ -3,14 +3,15 @@ version: '3.8'
 services:
   influxdb:
     image: influxdb:1.8
+    container_name: influxdb
     ports:
       - "8086:8086"
     volumes:
       - influxdb-data:/var/lib/influxdb
 
-  flask-app:
-    build:
-      context: .
+  nodejs-app:
+    build: .
+    container_name: nodejs_app
     ports:
```

Comment:
I'm also using `network` within docker, ex.
https://github.com/Gratheon/user-cycle/blob/main/docker-compose.dev.yml#L21

I don't remember now why though. But containers need to be in the same network to talk to each other